client/modules/IDE/components/Toolbar.jsx

```diff
@@ -86,23 +87,23 @@ class Toolbar extends React.Component {
             this.props.setTextOutput(true);
             this.props.setGridOutput(true);
           }}
-          aria-label="Play sketch"
+          aria-label={i18n.t('Toolbar.PlaySketchARIA')}
```

Comment:
I think we should use `withTranslation()` here. The [react-i18next](https://react.i18next.com/latest/using-with-hooks#translate-your-content) documentation doesn't mention the method you're using here at all which makes me concerned about using it.

All of these can be `this.props.t('...')`